app/services/tag_hashtag_data_source.rb

```diff
@@ -12,17 +12,14 @@ def self.type
     "tag"
   end
 
-  def self.tag_to_hashtag_item(tag, include_count: false)
+  def self.tag_to_hashtag_item(tag)
     tag = Tag.new(tag.slice(:id, :name, :description).merge(topic_count: tag[:count])) if tag.is_a?(
       Hash,
     )
 
     HashtagAutocompleteService::HashtagItem.new.tap do |item|
-      if include_count
-        item.text = "#{tag.name} x #{tag.topic_count}"
-      else
-        item.text = tag.name
-      end
+      item.text = tag.name
+      item.secondary_text = "x#{tag.topic_count}"
```

Comment:
We used to have a space before and after `x`. I believe the space before is still there, what about the after one?